docs/bcr-policies.md

```diff
@@ -18,12 +18,35 @@ Registry maintainers and module maintainers maintain the BCR. BCR maintainers ar
-  - Raise a PR to add a new module for the first time to the BCR.
-  - Raise a PR to change a presubmit.yml file for a module.
+- Review and accept contributions made by the community that align with the BCR contribution policy. Maintainers can merge any PR they have carefully reviewed and is passing CI. BCR maintainer attention is required when:
+  - A PR changes a module that has no existing maintainers: this requires the BCR maintainer to approve the PR using GitHub's PR review workflow.
+    - This includes when the PR adds an entirely new module, or if the module's sole existing maintainer is also the PR author (since a PR author cannot approve their own PR).
+  - A PR changes the presubmit.yml file for a module: this requires the BCR maintainer to apply the `presubmit-auto-run` label to unblock CI.
+  - A PR author is a first-time contributor to the BCR: this requires the BCR maintainer to click the "Approve and run" button to unblock certain GitHub actions.
 - Assess the health of the BCR by monitoring the BCR testing and serving infrastructure.
 - Identify and appoint module maintainers.
-- Triage issues in the BCR github repository.
+- Triage issues in the BCR GitHub repository.
+
+### BCR maintainer playbook
+
+Here are steps that a BCR maintainer should keep in mind when reviewing PRs:
+
+- When a PR adds an entirely new module to the BCR, perform a sanity check on its name.
+  - Ideally, the name is succinct and unambiguous. Overly generic names should be avoided unless it's well-known in the developer community. (If the project is hosted on GitHub, the number of stars would provide a good basic signal.) If necessary, ask the PR author to prepend their user or org to the module name.
+  - For example, a module named `package-utils` would be too generic. `mycompany-package-utils` would work instead.
+- When a PR adds to a module with no existing maintainers (including if the module is new), encourage the PR author to take on module maintainership.
+  - Modules with no maintainers always require BCR maintainer review, which adds to response times and toil.
+- When a module version uses overlays, remind PR authors to include `bazel_compatibility` of at least `7.2.1`.
+- When a PR contains C++ modules with BUILD patches or overlays, encourage usage of a build target with the same name as the module itself.
+  - This enables the shortform label syntax `@foo`, which is shorthand for `@foo//:foo`.
+  - For modules whose name start with `lib` (for example, `libfoo`), ask for a target named `foo` with an alias named `libfoo` pointing to `foo` ([example](https://github.com/bazelbuild/bazel-central-registry/pull/3821#discussion_r1959200504)).
+- When a PR contains C++ modules with BUILD patches or overlays, double check visibility specifications.
+  - PR authors sometimes forget `//visibility:public`, and this is often not caught unless the presubmit.yml has a test module.
```

Comment:
Could also mention that new modules with no upstream support for Bazel should also attempt to make as few targets public as possible to simplify future maintainers' lives.

Reply:
added a mention.